assets/config/tailwind.config.js

```diff
@@ -71,6 +71,18 @@ const green = {
     900: "#155148",
 }
 
+const fuschia= {
```

Comment:
I ended up not actually using these in this PR, but they're part of the palette, so I chose to leave 'em in, for future use.